subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/GradleExecuter.java

```diff
@@ -438,4 +438,9 @@
      * Execute the builds without adding the {@code "--stacktrace"} argument.
      */
     GradleExecuter withStacktraceDisabled();
+
+    /**
+     * Renders the welcome message users see upon first invocation of a Gradle distribution. By default the message is never rendered.
```

Comment:
The message will still only be rendered on first invocation, right? Maybe clarify this doc:

_Renders the welcome message upon first invocation of a Gradle distribution with a given gradle user home_

Reply:
> The message will still only be rendered on first invocation, right?
Correct.

done